.circleci/.config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2.1
```

Comment:
Is the naming of this file correct?

Error from circleci

`We couldn’t find a .circleci/config.yml `

Reply:
renamed to config.yml instead of .config.yml